generator/internal/dern.cc

```diff
@@ -0,0 +1,5 @@
+/// Returns `child` if tracing is not enabled.
```

Comment:
So what?

Reply:
Gah, thanks for catching this. `dern.*` is a pejorative version of my name that I use for temporary files. I did not mean to commit it. Removed.